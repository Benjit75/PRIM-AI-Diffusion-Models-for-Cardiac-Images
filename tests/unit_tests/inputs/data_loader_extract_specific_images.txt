{'data_names': ["test"], 'image_types': ["image_data"], 'image_names': ["ED", "ED_gt"]}