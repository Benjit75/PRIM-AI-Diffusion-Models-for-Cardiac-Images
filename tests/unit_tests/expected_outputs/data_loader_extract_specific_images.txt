testing/patient101/patient101_frame01.nii
testing/patient101/patient101_frame01_gt.nii
testing/patient102/patient102_frame01.nii
testing/patient102/patient102_frame01_gt.nii